keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -117,6 +121,18 @@ class SqlArtifactRepository(
     return objectMapper.writeValueAsString(details)
   }
 
+  private fun BuildMetadata.detailsAsJson(): String {
+    val details = objectMapper.convertValue<Map<String, Any?>>(this)
```

Comment:
Do you need to convert to a map first? I think write as string will do that for you.

Reply:
Done!